### DISCUSSION TITLE
Audio Source의 clip null 오류

### DISCUSSION THREAD

**Discussion by diddntjd99** [naverz/zepeto-studio-kor#14] [5. Q&A]
안녕하세요,
효과음을 넣기 위해서 오디오 소스를 사용하려 했는데, 인스펙트 창에서 Clip을 넣어줬는데도 clip이 null이라는 오류가 계속 발생합니다.
일반 유니티 프로젝트에서 똑같이 사용해보면 정상 작동하는데, 확인해주실 수 있으신가요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 현상이 최신 월드 1.1.5에서는 재현되지 않고 있습니다.
만약 관련 현상이 또 발생할 경우 말씀 주세요~
감사합니다.
